src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -423,15 +425,40 @@ void AutomatedRssDownloader::on_exportBtn_clicked()
 
 void AutomatedRssDownloader::on_importBtn_clicked()
 {
+    const QString ALL_SUPPORTED_RULES = "All supported formats (*.rssrules *.filters *.dat)";
+    const QString QBITTORRENT_RULES = "qBittorent (*.rssrules *.filters)";
+    const QString UTORRENT_RULES = "uTorrent (*.dat)";
+    const QString DIALOG_FILTER = ALL_SUPPORTED_RULES % ";;" % QBITTORRENT_RULES % ";;" % UTORRENT_RULES;
+
     // Ask for filter path
-    QString load_path = QFileDialog::getOpenFileName(this, tr("Please point to the RSS download rules file"), QDir::homePath(), tr("Rules list") + QString(" (*.rssrules *.filters)"));
+    QString selectedFilter = ALL_SUPPORTED_RULES;
+    QString load_path = QFileDialog::getOpenFileName(this, tr("Select RSS rules file"), QDir::homePath(), DIALOG_FILTER, &selectedFilter);
```

Comment:
There is no comprehensive explanation, but you can refer [here](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md) and remember that @glassez insists on correcting every line you touch :)